vue.config.js

```diff
@@ -133,39 +110,25 @@ const config = {
   devServer: {
     hot: true,
     https: useHttps,
-    disableHostCheck: true,
+    allowedHosts: 'all',
     headers: {
       "Access-Control-Allow-Origin": "*"
     }
   },
   runtimeCompiler: true,
   configureWebpack: {
+    performance: {
+      hints: false
+    },
+    devtool: false,
     resolve: {
       alias: {
-        'prosemirror-tables': path.join(__dirname, 'node_modules/prosemirror-tables/src/index.js'),
-        'prosemirror-state' : path.join(__dirname, 'node_modules/prosemirror-state/src/index.js'),
-        'prosemirror-view' : path.join(__dirname, 'node_modules/prosemirror-view/src/index.js'),
-        'prosemirror-transform' : path.join(__dirname, 'node_modules/prosemirror-transform/src/index.js')
-      },
-      modules: resolveModules
```

Comment:
We need to keep this. This allows building Twill with components importing dependencies installed in the host project's node modules.